### PR TITLE
dev-util/bear: add 3.1.3

### PR DESCRIPTION
**Pull request by sevz17**

### CODE REVIEW

**Comment by sevz17** [comment]
@thesamesam

File: dev-util/bear/bear-3.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+inherit cmake python-any-r1
+
+DESCRIPTION="Build EAR generates a compilation database for clang tooling"
+HOMEPAGE="https://github.com/rizsotto/Bear"
+SRC_URI="https://github.com/rizsotto/Bear/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~loong ~ppc64 ~riscv x86"
```
Comment (thesamesam) [inline dev-util/bear/bear-3.1.3.ebuild:16]: ❗

File: dev-util/bear/bear-3.1.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	virtual/pkgconfig
+	test? (
+		sys-devel/libtool
+		$(python_gen_any_dep '
+			dev-python/lit[${PYTHON_USEDEP}]
+		')
+	)
+"
+
+RESTRICT="!test? ( test )"
```
Comment (thesamesam) [inline dev-util/bear/bear-3.1.3.ebuild:44]: put this directly below IUSE

File: dev-util/bear/bear-3.1.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+S="${WORKDIR}/${P^}"
```
Comment (thesamesam) [inline dev-util/bear/bear-3.1.3.ebuild:46]: move directly below SRC_URI

File: dev-util/bear/bear-3.1.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_UNIT_TESTS="$(usex test)"
+		-DENABLE_FUNC_TESTS="$(usex test)"
+	)
+	cmake_src_configure
+}
+
+src_test() {
```
Comment (thesamesam) [inline dev-util/bear/bear-3.1.3.ebuild:66]: this whole src_test feels horrible and like it should be fixed properly but it's ok to do it another time

File: dev-util/bear/bear-3.1.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+RDEPEND="
+	>=dev-cpp/nlohmann_json-3.11.2:=
+	>=dev-db/sqlite-3.14:=
+	>=dev-libs/libfmt-9.1.0:=
+	dev-libs/protobuf:=
+	>=dev-libs/spdlog-1.11.0:=
+	>=net-libs/grpc-1.49.2:=
+"
+
```
Comment (thesamesam) [inline dev-util/bear/bear-3.1.3.ebuild:27]: add newline in DEPEND

File: dev-util/bear/bear-3.1.3.ebuild (hunk 1 above)
Comment (thesamesam) [reply dev-util/bear/bear-3.1.3.ebuild:16]: (use `pkgcheck scan --commits`)

Comment (sevz17) [reply dev-util/bear/bear-3.1.3.ebuild]: Ups!

**Comment by sevz17** [comment]
Done

**Pull request closed without merging**